ydb/core/tx/datashard/import_s3.cpp

```diff
@@ -96,6 +101,11 @@ class TS3Downloader: public TActorBootstrapped<TS3Downloader> {
             return std::make_pair(start, end);
         }
 
+        bool RestoreFromState(const TS3DownloadState& state, TString& error) override {
+            Y_UNUSED(state, error);
```

Comment:
Тут и далее:
```suggestion
        bool RestoreFromState(const TS3DownloadState& /* state */, TString& /* error */) override {
```

Или вообще без имен аргументов. Потом кто-нибудь начнет использовать эти аргументы, а `Y_UNUSED` убрать забудет и он будет висеть мертвым грузом.